src/components/10-atoms/link/react/DemoLink.jsx

```diff
@@ -15,7 +15,8 @@ const t = (scope, text) => {
 const DemoLink = () => {
   const [expanded, setExpanded] = useState(false);
 
-  const toggleExpand = () => {
+  const toggleExpand = ev => {
+    ev.preventDefault();
```

Comment:
Nice, so component users have an example using preventDefault from within - and we can point to that if questions arise.